Dockerfile.service-controller

```diff
@@ -8,7 +8,7 @@ COPY . .
 
 RUN make build-service-controller
 
-FROM node:12 AS console-builder
+FROM node:12.18 AS console-builder
```

Comment:
No, this is to prevent a failure in building the console. I can separate this out into a separate PR if you prefer. However it was needed to allow this PR to build under circle-ci.

Reply:
I am ok!
Note: we "also fixed" that issue changing the executor, so master CI is working now, without this change. Anyway, probably this change is a better solution...